packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -153,10 +154,19 @@ function ngGenerateTypescriptSDKFn(options: NgGenerateTypescriptSDKCoreSchematic
     let specContent!: string;
     if (URL.canParse(generatorOptions.specPath) && (new URL(generatorOptions.specPath)).protocol.startsWith('http')) {
       specContent = await (await fetch(generatorOptions.specPath)).text();
+      specContent = updateLocalRelativeRefs(specContent, path.dirname(generatorOptions.specPath));
     } else {
       const specPath = path.isAbsolute(generatorOptions.specPath) || !options.directory ?
         generatorOptions.specPath : path.join(options.directory, generatorOptions.specPath);
       specContent = readFileSync(specPath, {encoding: 'utf-8'}).toString();
+
+      if (path.relative(process.cwd(), specPath).startsWith('..')) {
+        // TODO would be better to create files on tree instead of FS
```

Comment:
Indeed. Can you please create an issue associated to it (and add its ref into the comment)?

Reply:
https://github.com/AmadeusITGroup/otter/issues/2078

Reply:
can you refer your issue in your `todo` comment?

Reply:
yes, the link is on the line below. I preferred the full link because it can be open directly from the IDE with ctrl+click